hugr-core/src/export.rs

```diff
@@ -44,10 +44,21 @@ struct Context<'a> {
     bump: &'a Bump,
     /// Stores the terms that we have already seen to avoid duplicates.
     term_map: FxHashMap<model::Term<'a>, model::TermId>,
+
     /// The current scope for local variables.
+    ///
+    /// This is set to the id of the smallest enclosing node that defines a polymorphic type.
+    /// We use this when exporting local variables in terms.
```

Comment:
Slightly nervous of "we use this when" comments because the far-away thing on the RHS can change and forget to update the comment...